apkid/rules/dex/obfuscators.yara

```diff
@@ -118,9 +118,14 @@ rule dexguard_c : obfuscator
     description = "DexGuard"
     url         = "https://www.guardsquare.com/en/products/dexguard"
     sample      = "de67161a8bd7ebcaa26c9661efd811375b59260924eb0dfd9436d3a47a1c31fe"
+    sample2     = "db11762886cc24bd4f938002f15f78680b512cf550d15c77b217fc5548b0c939"
 
   strings:
-    $dexguard_pkg = "guardsquare/dexguard/runtime"
+    $dexguard_pkg1 = "guardsquare/dexguard/runtime"
```

Comment:
Can these be null padded?

Reply:
This rule wasn't using null-padded strings but I will check it out as soon as I can get some free time.

Reply:
Ok, I added some fixes for the sample I was opening the MR for. Lets merge this, and I will update the others once I can find another sample